tests/manage/z_cluster/nodes/test_nodes_restart.py

```diff
@@ -77,6 +76,7 @@ def test_nodes_restart(
             pvc_factory, pod_factory, bucket_factory, rgw_bucket_factory
         )
 
+    @tier4b
```

Comment:
There is `tier4a` marker at L#27, wouldn't this test be considered as both tier4a and tier4b in that case?

Reply:
IMO I dont think the test will be considered as both tier4a and tier4b, Wont it override ?? @petr-balogh please correct me if i am wrong

Reply:
no it will not overwrite. Please remove it from class level and specify directly for each test case in that class.  This will really lead that the test will be tier4a and tier4b as well if not changed.

Reply:
Added them separately under different tests and removed it in class